### ISSUE TITLE
Gallery Block - Alt text missing on images, Edit icon missing

### ISSUE THREAD

**Issue by mikeicode** [Automattic/wp-calypso#36594]
On Simple sites there is no way to add Alt text to images in a Gallery block. Editing the images in the Media library to add the Alt text doesn't work either, the images in the Gallery block aren't updated.

If you remove the image from the Gallery block and then re-select it from the media library the Alt text is there, but that would require recreating all of your galleries.

Also, the edit icon is missing from the Gallery block, this icon allows you to click "Edit Gallery > Insert" which then does update the images with the Alt text you added, plus from this screen you can add Alt text if needed (this is the current setup on AT)

#### Steps to reproduce
1. Starting at URL: Simple Site
2. Add Gallery block 
3. Add images (there is no way to add Alt text on upload)
4. There is no way to then click "Edit gallery" to edit Alt text either.
5. Go to Media library (Site > Media)
6. Add Alt text to image
7. View published page, no Alt text on images.

#### What I expected

Need to have a way to edit Alt text for uploaded images to a Gallery block.

#### What happened instead

Currently on Simple sites you would need to delete the images from the Gallery block and add them back.

**Simple Site:**

ALT text on image is empty when first creating the Gallery, and after adding Alt text to the image in the media library:

![img](https://cld.wthms.co/3EK5JY+)

No way to edit the Gallery block to add Alt text to images:

![img](https://cld.wthms.co/H37MNZ+)

**AT Site**

Has the edit icon to edit gallery which allows you to edit Alt text:
![img]( https://cld.wthms.co/16E8is+)

#### Context / Source

Reported 15794315-hc tested on Simple and AT.
